src/css/tabs/osd.css

```diff
@@ -517,16 +527,31 @@ button {
   }
 }
 
-.tab-osd .grid-col.col4:first-child, .tab-osd  .grid-col.col4:last-child {
-    width: calc((100% - 390px) / 2);
+.tab-osd .grid-row {
+    justify-content: center;
+    gap: 7px;
+}
+
+.tab-osd .grid-row .elements {
+  flex: 1 1 auto;
+  width: 210px;
```

Comment:
180px seems enough on Windows / Linux

Reply:
EDIT: But 300px seems needed to keep these two on the same line:

![image](https://user-images.githubusercontent.com/8344830/168189318-577cd41a-df3c-4361-b1cc-c4dd2bacde4a.png)

Reply:
The same than before. The other PR will make it better:

https://github.com/betaflight/betaflight-configurator/pull/2920

but under very low resolutions/window sizes, something will not be perfect. The size is limited.